### DISCUSSION TITLE
进入游戏没有网络连接

### DISCUSSION THREAD

**Discussion by guozqwin** [pjy612/SteamManifestCache#659] [Question / Answer]
我能成功下载游戏，但是进入游戏后，没有网络连接，无法进行游戏，这是什么原因？

**Comment by pjy612** [comment]
正常来说 入库的游戏 不能联网很正常。
如果你是所有游戏都不能联机，包括自己买的。
那么有概率你用了 BTM的入库工具，推荐你更新一下。
或者 重装steam。
